app-editors/dav/dav-0.9.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Minimal console text editor"
+HOMEPAGE="https://github.com/atsb/dav-text"
+SRC_URI="https://github.com/atsb/dav-text/archive/refs/tags/dav-text-${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Why rename it at all?

Reply:
Yeah, don't do it. It's nicer if it matches upstream names as long as it's not generic for e.g. archival purposes and so on.